plugins/module_utils/omevv_utils/omevv_firmware_utils.py

```diff
@@ -655,3 +659,37 @@ def delete_baseline_profile(self, profile_id, vcenter_uuid):
         resp = self.omevv.invoke_request(
             "DELETE", BASELINE_PROFILE_URI.format(vcenter_uuid=vcenter_uuid) + "/" + str(profile_id))
         return resp
+
+
+class OMEVVFirmwareUpdate:
+    def __init__(self, omevv):
+        self.omevv = omevv
+
+    def update_cluster(self, vcenter_uuid, cluster_group_id, **kwargs):
```

Comment:
Add the kwargs sample input in docs of function

Reply:
Incorporated the comment